docs/spelling_wordlist.txt

```diff
@@ -1,3 +1,26 @@
+AES
+ASIO
+Asio
+Bitfields
+CDR
+Centric
+CMake
+Colcon
+GCM
+GMAC
+Github
+IDL
+IPv
+Intra
+Nagle
+PKI
+QoS
+Structs
+TCP
+Topologies
+TypeObject
+WString
+XTypes
```

Comment:
Watch put with this. Probably they are already there and it's just a by-product of the rebase. You can try to sort alphabetically to check